frontend/src/scenes/events/EventsTable.js

```diff
@@ -186,51 +188,54 @@ export function EventsTable({
                     Export
                 </Button>
             </Tooltip>
-            <Table
```

Comment:
Not sure about this <Card> around the table, takes up a lot of space and looks a bit odd I think?
![image](https://user-images.githubusercontent.com/1727427/97980031-7186e100-1dd0-11eb-8862-c079c1f2b1c4.png)

Reply:
Ah looks like it's on some other tables too. I do think it looks good on the setup/settings pages etc, but not sure about the tables.

Reply:
Agree with @timgl on the above